### ISSUE TITLE
Ignoring the .lazy modifier

### ISSUE THREAD

**Issue by cristian-frumusanu** [autoNumeric/vue-autoNumeric#26]
Hello.
Thanks for the great component!

I have identified one issue, though.

When using the `.lazy` modifier on `v-model`, the AutoNumeric component updates the model on the `input` event, rather than on the `change` event.

I have created a fiddle to illustrate the issue in which there is a standard input next to a Vue AutoNumeric component: https://jsfiddle.net/p462eLa9/1/

Please let me know your thoughts on this.

**Comment by AlexandreBonneau** [comment]
You are correct, I totally overlooked that [modifier](https://vuejs.org/v2/guide/forms.html#lazy)!

**Comment by cristian-frumusanu** [comment]
@AlexandreBonneau
I'm glad I could help.
Are you planning to release an update in the near future that would include the fix to this issue?
Thanks.

**Comment by AlexandreBonneau** [comment]
Yes, but it depends on your definition of 'near future' really :)
I'm pretty swamped for a few months now, so I'll see when I can get a closer look into that issue.

If you find the solution on your end, please feel free to create a PR!

**Comment by cristian-frumusanu** [comment]
@AlexandreBonneau 

I've done some digging around and I don't think it's possible without this:
https://github.com/vuejs/vue/issues/6914

**Comment by doucouredave** [comment]
Same issue. 
@AlexandreBonneau, have you planned to fix the issue?
@cristian-frumusanu , have you get a workaround?

**Comment by cristian-frumusanu** [comment]
> Same issue.
> @AlexandreBonneau, have you planned to fix the issue?
> @cristian-frumusanu , have you get a workaround?

@doucouredave 

I think this works:
```
<VueAutonumeric @change.native="(event) => { yourVariable = event.target.value }" />
```

**Comment by doucouredave** [comment]
Thanks for you reply @cristian-frumusanu,
what does mean `yourVariable`? is it `v-model`?

**Comment by cristian-frumusanu** [comment]
@doucouredave 
`yourVariable` is just the variable name. It could be `foo`, `bar`, whatever.

So, if you would normally have
```
<VueAutonumeric v-model="yourVariable" />
```

to make this work you have to write:
```
<VueAutonumeric @change.native="(event) => { yourVariable = event.target.value }" />
```

**Comment by rylanb** [comment]
I was seeing a warning from AutoNumeric about the model being undefined. So I added `:value="model"` to ^^
